### PR TITLE
Fix docs bug

### PR DESCRIPTION
**Pull request by nd-02110114**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ script:
 - if [ $TRAVIS_PYTHON_VERSION == '3.7' ]; then
   find ./deepchem -name "*.py" ! -name '*load_dataset_template.py' | xargs python -m doctest -v; fi
 - bash devtools/travis-ci/test_format_code.sh
+- if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then
```
Comment (nd-02110114) [inline .travis.yml:35]: Check the docs build

File: docs/moleculenet.rst [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
+5. Ask a member of the technical steering committee to add your .tar.gz or .zip file 
+   to the DeepChem AWS bucket. Modify your load function to pull down the dataset from AWS.
 
-6. Submit a [WIP] PR (Work in progress pull request) following the PR `template <https://github.com/deepchem/deepchem/blob/master/docs/molnet_pr_template.md>`_.  
+6. Submit a [WIP] PR (Work in progress pull request) following the PR `template`_.
 
-Load Dataset Template
----------------------
-
-.. autofunction:: deepchem.molnet.load_function.load_dataset_template.load_mydataset
```
Comment (nd-02110114) [inline docs/moleculenet.rst:28]: This line broke the docs build. `load_dataset_template` raise error when being loaded.

File: .travis.yml [hunk 3]
```diff
@@ -32,6 +32,8 @@ script:
 - if [ $TRAVIS_PYTHON_VERSION == '3.7' ]; then
   find ./deepchem -name "*.py" ! -name '*load_dataset_template.py' | xargs python -m doctest -v; fi
 - bash devtools/travis-ci/test_format_code.sh
+- if [ $TRAVIS_PYTHON_VERSION == '3.7' ]; then
+  cd docs && pip install -r requirements.txt && make clean html && cd ..; fi
```
Comment (nd-02110114) [inline .travis.yml:36]: Check the docs build

Comment (rbharath) [reply .travis.yml:36]: Good addition!

File: docs/moleculenet.rst [hunk 4]
```diff
@@ -197,3 +201,13 @@ UV Datasets
...
+
+
+.. _`datasets already available in MolNet`: http://moleculenet.ai/datasets-1
+.. _`Contribution guidelines`: https://github.com/deepchem/deepchem/blob/master/CONTRIBUTING.md
+.. _`issue`: https://github.com/deepchem/deepchem/issues
+.. _`deepchem.molnet.load_function`: https://github.com/deepchem/deepchem/tree/master/deepchem/molnet/load_function
+.. _`deepchem.molnet.load_function.load_dataset_template`: https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/load_function/load_dataset_template.py
+.. _`deepchem.molnet.defaults`: https://github.com/deepchem/deepchem/tree/master/deepchem/molnet/defaults.py
+.. _`deepchem.molnet.__init__.py`: https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/__init__.py
+.. _`template`: https://github.com/deepchem/deepchem/blob/master//molnet_pr_template.md
```
Comment (rbharath) [inline docs/moleculenet.rst:213]: I think there's a typo here? Should be https://github.com/deepchem/deepchem/blob/master/.github/PULL_REQUEST_TEMPLATE/molnet_pr_template.md

Comment (nd-02110114) [reply docs/moleculenet.rst]: I fixed!

**Comment by rbharath** [comment]
Looks good!

**Pull request merged**